src/main/java/org/kainos/ea/api/AuthService.java

```diff
@@ -0,0 +1,44 @@
+package org.kainos.ea.api;
+
+import io.jsonwebtoken.Claims;
+import org.kainos.ea.cli.Credential;
+import org.kainos.ea.client.ActionFailedException;
+import org.kainos.ea.client.AuthenticationException;
+import org.kainos.ea.client.ValidationException;
+import org.kainos.ea.core.CredentialValidator;
+import org.kainos.ea.db.AuthDao;
+import org.kainos.ea.db.DatabaseConnector;
+
+public class AuthService {
+    private  final DatabaseConnector databaseConnector;
+    private final AuthDao authDao;
+    CredentialValidator credentialValidator = new CredentialValidator();
```

Comment:
You pass in the `CredentialValidator` as part of the constructor, does it also need to be initialised here as well?